app/src/main/java/com/infomaniak/mail/ui/main/menuDrawer/items/DividerItem.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Infomaniak Mail - Android
+ * Copyright (C) 2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.mail.ui.main.menuDrawer.items
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.viewbinding.ViewBinding
+import com.infomaniak.mail.R
+import com.infomaniak.mail.databinding.ItemMenuDrawerDividerBinding
+
+object DividerItem {
+
+    @Suppress("MayBeConstant")
+    val viewType = R.layout.item_menu_drawer_divider
```

Comment:
The Android doc says to use the ID : _"Consider using id resources to uniquely identify item view types."_

Reply:
<img width="677" alt="image" src="https://github.com/user-attachments/assets/37d2c73f-b252-464d-b75f-5e981343712a">

Reply:
What a strange way to go about it. I wonder what's the reasoning behind this because this doesn't feel like the correct way of doing things. There's less "hacky" ways to get unique values.

I've only found a few people saying this makes it so inside `override fun onCreateViewHolder(parent: ViewGroup, viewType: Int)` you have the layout id thanks to the `viewType` argument and can directly inflate your layout using it. But we don't really care about it as we use bindings anyway. I'd be curious to know if @sirambd has a clue about the reasoning behind that one

Reply:
Apparently it was useful because back when it got added this was an easy way of having a unique id. So let's keep this I guess